index.html

```diff
@@ -2287,31 +2302,14 @@ <h1>Speakers</h1>
                                             <a target="default" href="https://gultsch.de/"><i class="icon fa fa-external-link"></i></a>
                                             <a target="default" href="https://twitter.com/iNPUTmice"><i class="icon social_twitter"></i></a>
                                             <a target="default" href="https://github.com/inputmice/"><i class="icon fa fa-github"></i></a>
+                                            <a target="default" href="https://twitter.com/road2stat?lang=en"><i class="icon social_twitter"></i></a>                                          
                                         </div>
                                     </div>
                                 </div>
                                 <span class="speaker-name">Daniel Gultsch</span>
                                 <span>Conversations</span>
                             </div>
                         </div>
-                        <div class="col-md-3 col-sm-6 speaker-column">
```

Comment:
Refer #111

Reply:
what are you talkking about?

Reply:
He's saying to look into that specific issue @samyakbvs 
You have conflicted that issue.

Reply:
@samyakbvs  Don't delete anything with respect to Carsten Haitzler part and Masakazu Takasu part. You can add more profile links if you want.